test/unit/mdc-menu/mdc-menu.test.js

```diff
@@ -26,10 +26,10 @@ import bel from 'bel';
 import td from 'testdouble';
 import domEvents from 'dom-events';
 
-import {MDCMenu, MDCMenuFoundation} from '../../../packages/mdc-menu/index';
-import {Corner} from '../../../packages/mdc-menu-surface/constants';
-import {MDCListFoundation} from '../../../packages/mdc-list/index';
-import {MDCMenuSurfaceFoundation} from '../../../packages/mdc-menu-surface/foundation';
+import {MDCMenu, MDCMenuFoundation} from '../../../packages/mdc-menu/index.ts';
```

Comment:
I don't think adding the `.ts` extension is necessary. It should be resolved by webpack.

Reply:
Removed